### DISCUSSION TITLE
Exec permission in Android 10+

### DISCUSSION THREAD

**Discussion by RohitVerma882** [termux/termux-packages#19159] [Q&A] [answered]
How jvdroid able to execute binarys from app filesdir with published & updates to playstore.?

He uses something linker/linker64 to exec dynamic binarys

Why termux can not do that?

**Comment by RohitVerma882** [comment]
![Screenshot_2024-02-05-10-54-34-51_d4003123f13cd1833dbcae05882554f0](https://github.com/termux/termux-packages/assets/56387351/894cc931-3ea6-4361-a421-1c742b253a25)

**Comment by twaik** [comment, marked as answer]
https://github.com/termux/termux-exec/pull/24
